lib/trino-filesystem-alluxio/src/main/java/io/trino/filesystem/alluxio/AlluxioFileSystemInputFile.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.filesystem.alluxio;
+
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.URIStatus;
+import alluxio.exception.AlluxioException;
+import alluxio.exception.FileDoesNotExistException;
+import alluxio.exception.runtime.NotFoundRuntimeException;
+import alluxio.grpc.OpenFilePOptions;
+import io.trino.filesystem.Location;
+import io.trino.filesystem.TrinoInput;
+import io.trino.filesystem.TrinoInputFile;
+import io.trino.filesystem.TrinoInputStream;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.Optional;
+
+import static io.trino.filesystem.alluxio.AlluxioUtils.convertToAlluxioURI;
+import static java.util.Objects.requireNonNull;
+
+public class AlluxioFileSystemInputFile
+        implements TrinoInputFile
+{
+    private final Location location;
+
+    private final FileSystem fileSystem;
+
+    private final String mountRoot;
+    private Optional<Instant> lastModified;
+
+    private Long length;
+
+    private URIStatus status;
```

Comment:
The way we organize this in general is to keep all `final` member variables together without newline, then newline, followed by all the remaining variables without newline.
Please implement this in all the classes